example/advanced/server-flex/advanced_server_flex.cpp

```diff
@@ -614,13 +521,53 @@ class http_session
         }
 
         // Send the response
-        handle_request(*doc_root_, parser_->release(), queue_);
+        enqueue(handle_request(*doc_root_, parser_->release()));
```

Comment:
Can't say I'm a fan of this name

Reply:
append, push_back, send, maybe_send, add_response.

`do_write` is also perfectly fine. `enqueue` is above the target reading comprehension level. Or how about `maybe_write`?